tests/integration/components/frost-subcontext-actions-test.js

```diff
@@ -0,0 +1,53 @@
+import {expect} from 'chai'
+import {setupComponentTest} from 'ember-mocha'
+import hbs from 'htmlbars-inline-precompile'
+import {beforeEach, describe, it} from 'mocha'
+import sinon from 'sinon'
+
+describe('Integration | Component | frost subcontext actions', function () {
+  setupComponentTest('frost-subcontext-actions', {
+    integration: true
+  })
+
+  describe('when onClick is given and button is clicked', function () {
+    let dispatchHandler, handler
+    beforeEach(function () {
+      dispatchHandler = sinon.spy()
+      handler = sinon.spy()
+      this.setProperties({dispatchHandler})
+      this.setProperties({handler})
+      this.render(hbs`
+      {{frost-subcontext-actions
+        hook='subcontext-actions'
+        subcontentActions=(array
+          (hash
+            label='Do something'
+            action=(action handler)
+            icon=(hash
+              pack='frost'
+              name='round-add'
+            )
+          )
+          (hash
+            label='Action type'
+            action=(hash type="string")
+            icon=(hash
+              pack='frost'
+              name='round-add'
+            )
+          )
+        )
+        onDispatch=dispatchHandler
+      }}
+
+      `)
+
+      this.$('button').click()
```

Comment:
I'd like to see a `hook` (ember-hook) used here - also we need to test both types of actions - the one that relies on dispatch and the other with the explicit action handler.

I'm actually thinking in the case where the action handler is explicit it probably makes more sense to use `onClick` as the property than `action` - thoughts?